eclipse-theia/theia
# Provide all selected URIs for command execution

#### What it does

Closes https://github.com/eclipse-theia/theia/pull/10198

VSCode provides in addition to the URI of the last selected file in the explorer also an array of all selected URIs. This change aligns Theia to this behavior.

#### How to test

1. Install the [test extension](https://github.com/msujew/theia-test-10198/raw/main/theia-test-10198-0.0.1.vsix) ([source](https://github.com/msujew/theia-test-10198/blob/main/theia-test-10198-0.0.1.vsix))
2. Select multiple/one file in the file explorer
3. Use the `Show URIs` command in the context menu of the file explorer
4. It should show the same output as if installed in vscode, including all selected URIs

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
